MIGRATIONS.unreleased.md

```diff
@@ -8,4 +8,7 @@ User-facing changes are documented in the [changelog](CHANGELOG.released.md).
 ## Unreleased
 [Commits](https://github.com/scalableminds/webknossos/compare/24.05.0...HEAD)
 
+- Edited the annotations table to add the option to locked explorative annotations. Migration 114 needs to be executed to update the database schema.
```

Comment:
I think, we don't mention the reason for a migration usually. cc @fm3

Reply:
I agree, please have a look in Migrations.released.md to see how we listed the sql migrations.